assets/base.css

```diff
@@ -622,7 +622,8 @@ h6:empty {
   margin-left: 1rem;
 }
 
-.link[role="link"]:not([href]) {
+.link[role="link"]:not([href]),
+.full-unstyled-link[role="link"]:not([href]) {
   cursor: not-allowed;
 }
```

Comment:
Could we do something like `a:not([href])` or would that not account for every case?

Reply:
Hi @bacounts, adding `pointer-events: none;` below would actually overwrite `cursor: not-allowed;`.

Using `cursor: not-allowed;` provides a visual indication that a link is disabled and in this particular case, a placeholder link. We also wanted to follow the cursor behaviour for disabled buttons seen on some of our other sections like the Multicolumn section:

![image](https://user-images.githubusercontent.com/28404165/152869901-b7750ef4-7497-449d-8a76-54fc6f2e41b5.png)

You can read more about those changes in #730